.github/workflows/changeset-status.yml

```diff
@@ -41,14 +41,22 @@ jobs:
           # a changeset is needed or not.
           fetch-depth: 0
 
-      - name: Start check
+      - name: Start status check
+        id: start_status_check
         env:
-          GH_TOKEN: ${{ steps.app-token.outputs.token }}
+          GH_TOKEN: ${{ steps.app_token.outputs.token }}
         run: |
-          gh api --method POST repos/:owner/:repo/check-runs \
-            --header "Accept: application/vnd.github+json" \
-            --field head_sha=${{ github.event.workflow_run.head_sha }} \
-            --field name="${{ github.workflow }}" \
+          check_run_id=$(
+            gh api repos/:owner/:repo/check-runs \
+              --method POST \
+              --header "Accept: application/vnd.github+json" \
+              --field head_sha=${{ github.event.workflow_run.head_sha }} \
+              --field name="${{ github.job }}" \
```

Comment:
dit is de naam die te zien is in de PR, had er github.workflow van gemaakt, maar github.job is beter want die hebben we ook in de ruleset